### PR TITLE
Prepare release 0.9.0

### PR DESCRIPTION
**Pull request by robin-tribe29**
## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

See PRs in next section.

## What is the new behavior?

- #140 
- #138
- #126 
- #146 
- #147

### CODE REVIEW

File: roles/server/defaults/main.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  - CentOS 8
-  - RHEL 7
-  - RHEL 8
+  - Debian-11
+  - Ubuntu-20
+  - Ubuntu-22
+  - CentOS-7
+  - CentOS-8
+  - RedHat-7
+  - RedHat-8
```
Comment (robin-tribe29) [inline roles/server/defaults/main.yml:9]: @diademiemi can you tell, if replacing `RHEL` with `RedHat` will break the role for you?

File: roles/server/vars/RedHat.yml [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+  RedHat-7:
+    - firewalld
+    - epel-release
+    - libsemanage-python
+    - yum-plugins-core
+  RedHat-8:
+    - firewalld
+    - epel-release
+    - python3-libsemanage
+    - dnf-plugins-core
```
Comment (robin-tribe29) [inline roles/server/vars/RedHat.yml:25]: @diademiemi can you check, if this works for you?

File: roles/server/tasks/main.yml [hunk 3]
```diff
@@ -4,35 +4,18 @@
...
-- name: "Install Checkmk Prerequisites."
-  become: true
-  ansible.builtin.package:
-    name: "{{ item }}"
-    state: present
-  loop:
-    - "{{ checkmk_server_prerequisites }}"
-  tags:
-    - install-package
-    - install-prerequisites
```
Comment (robin-tribe29) [inline roles/server/tasks/main.yml:29]: @mto79 can you explain, why you moved the tasks to the OS playbooks? I missed this in your PR I think.

File: roles/server/defaults/main.yml (hunk 1 above)
Comment (diademiemi) [reply roles/server/defaults/main.yml:9]: I'll spin up a RHEL VM and will try this out

Comment (diademiemi) [reply roles/server/defaults/main.yml:9]: RHEL doesn't appear anywhere, it does seem to be `RedHat`. So the `RHEL` lines can be removed

File: roles/server/tasks/main.yml (hunk 3 above)
Comment (robin-tribe29) [reply roles/server/tasks/main.yml:29]: Actually this breaks the role. I will revert it, if no light is shed on the change @mto79.

Comment (mto79) [reply roles/server/tasks/main.yml:29]: Sorry Robin on vacation at the moment, have an alternative for this in mine codebase. This has to be reverted indeed!

File: roles/server/vars/RedHat.yml (hunk 2 above)
Comment (diademiemi) [reply roles/server/vars/RedHat.yml:25]: Doesn't seem like epel-release is available in RHEL. I'll look at mto79s codebase and see how he did this differently. I'll create a PR with the changes to try to get it working on RHEL 7 and RHEL 8.

Comment (robin-tribe29) [reply roles/server/vars/RedHat.yml:25]: I think `epel` should be available for RHEL, just not sure if you enable it just like with CentOS or similar.

Comment (diademiemi) [reply roles/server/vars/RedHat.yml:25]: It should be yeah. But regardless I've heard the role breaks at others points on RHEL systems so I'll take a look at that today and PR

File: roles/server/tasks/main.yml (hunk 3 above)
Comment (robin-tribe29) [reply roles/server/tasks/main.yml]: I reverted the change, feel free to have another look. Will probably merge this next week.

File: roles/server/vars/RedHat.yml (hunk 2 above)
Comment (diademiemi) [reply roles/server/vars/RedHat.yml]: https://github.com/tribe29/ansible-collection-tribe29.checkmk/pull/146 includes necessary changes

**Pull request merged**